app/controllers/conversations_controller.rb

```diff
@@ -21,6 +21,7 @@ def index
       @message = @conversation.messages.new
     end
     @mobile = params[:mobile] unless params[:mobile].nil?
+    @conversation_page = true
```

Comment:
Nem szükséges eltárolni ezt változóban, a `controller_name` és `action_name` metódusok elérhetőek a view-ből.
Plusz látom, hogy a `@index`-et is hasonlóan használjátok sok helyen, azt már ki lehetne szervezni egy helperbe mondjuk `index_page?` néven.